change/@internal-react-composites-2a3ec15a-0e73-4cb7-81f6-b81ad9e6fc7f.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "prerelease",
+  "packageName": "@internal/react-composites",
+  "email": "2684369+JamesBurnside@users.noreply.github.com",
```

Comment:
That email looks fishy to me.

Reply:
It's this setting: 
![image](https://user-images.githubusercontent.com/2684369/126350736-1114f7a8-ef14-41e0-b000-be50d15536c7.png)
(+ setting my local gitconfig to be this address also)